api/src/pcapi/core/fraud/ubble/models.py

```diff
@@ -99,10 +43,29 @@ class UbbleIdentificationAttributes(UbbleIdentificationObject):
     webhook: str
 
 
+class UbbleReasonCode(UbbleIdentificationObject):
+    type: str = pydantic.Field(alias="type")
+    id: str = pydantic.Field(alias="id")
+
+
+class UbbleReasonCodesMeta(UbbleIdentificationObject):
+    count: int = pydantic.Field(0, alias="count")
+
+
+class UbbleReasonCodes(UbbleIdentificationObject):
+    meta: UbbleReasonCodesMeta
+    data: list[UbbleReasonCode]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Supprimer meta
On utilise pas meta ou count par la suite